themes/bootstrap3/templates/search/facet-list.phtml

```diff
@@ -28,38 +26,43 @@
     <label><?=$this->translate('Sort') ?></label>
     <div class="btn-group">
       <?php foreach ($this->sortOptions as $key => $sort): ?>
-        <a href="<?=$urlBase . '&amp;facetpage=1&amp;facetsort=' . urlencode($key) ?>" class="btn btn-default js-facet-sort<?php if ($this->sort == $key): ?> active<?php endif; ?>" data-sort="<?=$key ?>" data-lightbox-ignore><?=$this->translate($sort) ?></a>
+        <a href="<?=$this->escapeHtmlAttr($urlBase . '&facetpage=1&facetsort=' . urlencode($key) . '&contains=' . urlencode($contains)) ?>" class="btn btn-default js-facet-sort<?php if($this->sort == $key): ?> active<?php endif; ?>" data-sort="<?=$key ?>" data-lightbox-ignore><?=$this->translate($sort) ?></a>
       <?php endforeach; ?>
     </div>
+
+    <div class="btn-group facet-lightbox-filter hidden">
+      <form>
+        <fieldset>
+          <?php if ($this->results->getParams()->supportsFacetFiltering($this->facet) === true): ?>
+            <legend><?=$this->transEsc('Filter') ?></legend>
```

Comment:
Why is this `legend` instead of `label class="control-label" for="..."`?

Reply:
I think this goes back to [this discussion](https://github.com/vufind-org/vufind/pull/2991#discussion_r1335806040) in this PR from ~sep/oct 23 that we should use fieldset/legend here.

Reply:
Right, ok. Then it just needs proper classes for style and alignment. Perhaps control-label would work for legend too?

Reply:
Also regarding the vertical centering of the button in the other comment, i think it would be better to use some native bootstrap3 classes like form-inline with form-groups instead and also change the sort options so that the whole form will look more normalized and the vertical alignment also works.

Reply:
No objection here. If anything is non-standard, it's probably my fault for throwing the reset button in without entirely knowing what I was doing. Feel free to clean up if you see a path forward. :-)

Reply:
As noted in my other comment, things look visually better now. However, I see that the fieldset and legend have been removed... yet I think we still need those for the accessibility reasons that led to them being introduced originally: without them, to a screen reader, there is no way to differentiate between the top search box submit and reset buttons and the facet filter submit and reset buttons, since they are identical. The purpose of the fieldset was to scope those buttons so it was clear that they were associated with the facet filter form. If we can't use fieldset/legend, then I think we need to introduce separate descriptive labels for those buttons so their purposes are clear and distinct.

Reply:
According to [this article](https://www.tpgi.com/fieldsets-legends-and-screen-readers-again/) i added `role="group"` and `aria-labelledby` which seems to fulfil the same purpose & avoids breaking the form's layout as it is right now.

Reply:
Ahh, thank you, I see now!